tasks/testapi.yml

```diff
@@ -16,39 +17,48 @@
   register: docker_response
   when:
     - testapi_deploy
+
+
 - name: Waiting TestAPI
   pause:
     seconds: '{{ testapi_wait }}'
   when:
     - docker_response.changed
+
 - name: Checking if pod is already registered
   uri:
     url: '{{ testapi_url }}/pods/{{ project }}'
     status_code:
       - 200
       - 404
+    use_proxy: no
```

Comment:
I would consider it must be removed. We could consider that you may uuse proxy to reach an external TestDB (OPNFV for instance)

Reply:
ok